src/components/Icon/index.js

```diff
@@ -93,7 +90,7 @@ Icon.propTypes = {
 
 Icon.defaultProps = {
   size: '3',
-  color: 'blue',
+  color: first(colors),
```

Comment:
Changed default icon color to `dark-gray` which works on dark ($base) and white backgrounds.